profile/Client/profileClient.go

```diff
@@ -425,8 +452,30 @@ func generateRowsFromData(data []pb.Log, Operation string) []table.Row {
 			ColumnCount:         frequency.freq,
 			ColumnTimestamp:     frequency.time,
 		})
+		jsondata = append(jsondata, Profile{
+			Namespace:     r.Namespace,
+			ContainerName: r.ContainerName,
+			Process:       r.Process,
+			Resource:      r.Resource,
+			Result:        r.Result,
+			Count:         frequency.freq,
+			Time:          frequency.time,
+		})
 		s.rows = append(s.rows, row)
 	}
+
+	if o1.Save {
+		if Operation == "File" {
+			convertToJson("File", jsondata)
+		} else if Operation == "Process" {
+			convertToJson("Process", jsondata)
+		} else if Operation == "Network" {
+			convertToJson("Network", jsondata)
+		} else if Operation == "Syscall" {
+			convertToJson("Syscall", jsondata)
+		}
+	}
```

Comment:
How does this work? Will it only save when we are watching it?

Reply:
yup, if `--save` flag is there, then only it will watch and save it

Reply:
Not that, does it save a single operation at a time?

Reply:
Nope, it saves all of them together.

Reply:
ref for generating table: https://github.com/kubearmor/kubearmor-client/blob/main/profile/Client/profileClient.go#L230

Reply:
Thanks, Gotit.